packages/ui/ViewManager/tests/View-specs.js

```diff
@@ -1,250 +1,192 @@
-import {mount} from 'enzyme';
+import {createRef} from 'react';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import View from '../View';
 
 import {MockArranger} from './test-utils';
 
 describe('View', () => {
+	test('should render its child when neither enteringProp or childProps is specified', () => {
+		render(
+			<View duration={1000}>
+				<span data-testid="span" />
+			</View>
+		);
 
-	test(
-		'should render its child when neither enteringProp or childProps is specified',
-		() => {
-			const subject = mount(
-				<View duration={1000}>
-					<span />
-				</View>
-			);
+		const actual = screen.getByTestId('span');
 
-			const expected = 1;
-			const actual = subject.find('span').length;
+		expect(actual).toBeInTheDocument();
+	});
 
-			expect(actual).toBe(expected);
-		}
-	);
+	test('should pass the value of entering to its child in enteringProp', () => {
+		render(
+			<View duration={1000} enteringProp="data-entering">
+				<span data-testid="span" />
+			</View>
+		);
 
-	test(
-		'should pass the value of entering to its child in enteringProp',
-		() => {
-			const subject = mount(
-				<View duration={1000} enteringProp="data-entering">
-					<span />
-				</View>
-			);
+		const actual = screen.getByTestId('span');
+
+		expect(actual).toHaveAttribute('data-entering', 'true');
+	});
+
+	test('should pass enteringProp as false for an appearing view', () => {
+		// Views visible on mount are "appearing" and shouldn't perform "entering" logic like
+		// deferring children rendering
+		render(
+			<View duration={1000} enteringProp="data-entering" appearing>
+				<span data-testid="span" />
+			</View>
+		);
 
-			const expected = true;
-			const actual = subject.find('span').prop('data-entering');
+		const actual = screen.getByTestId('span');
 
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should pass enteringProp as false for an appearing view',
-		() => {
-			// Views visible on mount are "appearing" and shouldn't perform "entering" logic like
-			// deferring children rendering
-			const subject = mount(
-				<View duration={1000} enteringProp="data-entering" appearing>
+		expect(actual).toHaveAttribute('data-entering', 'false');
+	});
+
+	describe('imperative API without arranger', () => {
+		test('should call callback immediately for "stay"', () => {
+			const spy = jest.fn();
+			const ref = createRef();
+			render(
+				<View duration={16} ref={ref}>
 					<span />
 				</View>
 			);
+			ref.current.componentWillStay(spy);
 
-			const expected = false;
-			const actual = subject.find('span').prop('data-entering');
-
-			expect(actual).toBe(expected);
+			expect(spy).toHaveBeenCalledTimes(1);
```

Comment:
create a const expected = 1; for this test and the next two and replace in `expect`